app-connector-client/server/routes/connector.ts

```diff
@@ -107,25 +105,28 @@ function createInfo() {
     return null
 }
 
-function runOpenSSL(subject) {
+function runOpenSSL(subject: any) {
 
     LOGGER.debug("Creating CSR using subject %s", subject)
-    var privateKey = fs.readFileSync(keyFile, 'utf8')
-    var pk = forge.pki.privateKeyFromPem(privateKey)
-    var publickey = forge.pki.setRsaPublicKey(pk.n, pk.e)
+    let privateKey = fs.readFileSync(keyFile, 'utf8')
+    let pk: any = pki.privateKeyFromPem(privateKey)
+    //let publicKey = pki.setRsaPublicKey(pk.n, pk.e)

```

Comment:
what happens to the commented lines? Either the code change works and you remove the comments or it does not work and the PR cannot be merged.